x-pack/plugins/lens/public/datasources/form_based/form_based.tsx

```diff
@@ -261,7 +261,7 @@ export function getFormBasedDatasource({
           ...state,
           layers: {
             ...newLayers,
-            [layerId]: blankLayer(state.currentIndexPatternId, state.layers[layerId].linkToLayers),
+            [layerId]: blankLayer(state.currentIndexPatternId, state.layers[layerId]?.linkToLayers),
```

Comment:
ℹ️ This fixes a bug that exists in main, when trying to clear a layer on text based languages. I decided to fix it here.